STEERING-COMMITTEE.md

```diff
@@ -162,13 +120,157 @@ There are two ways to raise issues to the steering committee for decision:
+vote, are not running for Steering in that election, and can make a public
+promise of impartiality.  They will be responsible for:
+
+* Making all announcements associated with the election
+* Preparing and distributing electronic ballots
+* Judging exception requests
+* Assisting candidates in preparing and sharing statements
+* Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote
+
+Anyone who has at least 50 contributions in the last 12 months is eligible to
+vote in the Steering election. Contributions are defined as opening PRs,
+reviewing and commenting on PRs, opening and commenting on issues, writing
+design docs, commenting on design docs, helping people on slack, participating
+in working groups, and other efforts that help advance Knative the project.
+
+[This
+dashboard]((https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year)
+shows only GitHub based contributions and does not capture all the contributions
+we value. We expect this metric not to capture everyone who should be eligible
+to vote. If a community member has had significant contributions over the past
+year but is not captured in the [stats.knative.dev
+dashboard](https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year),
+they will be able to submit an exception form to the Elections Officer(s) who
+will then review and determine whether this member should be marked as an
+exception.  All exceptions, and the reasons for them, will be recorded in a log
+that will be available to Steering and the TOC.
+
+All eligible voters will be captured at
+knative/community/steering-elections/$YEAR/voters.md and the voters’ guide will
+be captured at knative/community/steering-elections/$YEAR/README.md, similar to
+the kubernetes election process and identical to the TOC election process.
+
+We are committed to an inclusive process and will adapt future eligibility
+requirements based on community feedback.
+
+### Candidate Eligibility
+
+Community members must be eligible to vote in order to stand for election (this
+includes voters who qualify for an exception). Candidates may self-nominate or
+be nominated by another eligible member. The approximate time commitment of a
+Steering member is around 5 hours per week.
+
+To run for a Contributing seat, a candidate must additionally be at least a
+project Member as defined in Roles.md.
+
+To run for an End-User seat, a candidate must be employed by, or own, an
+organization that does not sell products or services derived from Knative.
+Staff with in-house Knative deployments, as well as application developers who
+create applications that run on Knative, are eligible (and encouraged to run)
+for these seats.  Additionally, the candidate must participate in the
+knative-users mailing list and/or community-facing Knative Slack channels.
```

Comment:
It's a little odd to have this in combination with the "must be able to vote" requirement.

I'm willing to give it a try, but it might make sense to specifically call out that candidates here should feel welcome to apply for an exception.

Reply:
agree, this feels a bit hard to qualify for an end-user seat: you need to use, but not sell and contribute..

Reply:
Agree that it's a little odd. Thinking about how to phrase this.